hdfs_datanode/metrics.csv

```diff
@@ -0,0 +1,12 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+hdfs_datanode.dfs_remaining,gauge,,byte,,The remaining disk space left in bytes,0,hdfs_datanode,dfs rem
+hdfs_datanode.dfs_capacity,gauge,,byte,,Disk capacity in bytes,0,hdfs_datanode,dfs cap
+hdfs_datanode.dfs_used,gauge,,byte,,Disk usage in bytes,0,hdfs_datanode,dfs usd
+hdfs_datanode.cache_capacity,gauge,,byte,,Cache capacity in bytes,0,hdfs_datanode,cache cap
+hdfs_datanode.cache_used,gauge,,byte,,Cache used in bytes,0,hdfs_datanode,cache usd
+hdfs_datanode.num_failed_volumes,gauge,,,,Number of failed volumes,0,hdfs_datanode,faild vols
+hdfs_datanode.last_volume_failure_date,gauge,,,,Date the last volume failed,0,hdfs_datanode,lst faild vol

```

Comment:
I guess this metric should be the number of seconds/minutes/hours since the last volume failed, not an actual date right?